app/serializers/institution_profile_serializer.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+class InstitutionProfileSerializer < ActiveModel::Serializer
+  include OptionalSerializedAttributes
+
+  attribute :institution
+  attribute :facility_code
+  attribute :institution_type_name, key: :type
+  attribute :city
+  attribute :state
+  attribute :zip
+  attribute :country
+  attribute :bah
+  attribute :cross
+  attribute :ope
+  attribute :pred_degree_awarded
+  attribute :locale
+  attribute :gibill, key: :student_count
+  attribute :undergrad_enrollment
+  attribute :yr
+  attribute :student_veteran
+  attribute :student_veteran_link
+  attribute :poe
+  attribute :eight_keys
+  attribute :dodmou
+  attribute :sec_702
+  attribute :vetsuccess_name
+  attribute :vetsuccess_email
+  attribute :credit_for_mil_training
+  attribute :vet_poc
+  attribute :student_vet_grp_ipeds
+  attribute :soc_member
+  attribute :va_highest_degree_offered
+  attribute :retention_rate_veteran_ba
+  attribute :retention_all_students_ba
+  attribute :retention_rate_veteran_otb
+  attribute :retention_all_students_otb
+  attribute :persistance_rate_veteran_ba
+  attribute :persistance_rate_veteran_otb
+  attribute :graduation_rate_veteran
+  attribute :graduation_rate_all_students
+  attribute :transfer_out_rate_veteran
+  attribute :transfer_out_rate_all_students
+  attribute :salary_all_students
+  attribute :repayment_rate_all_students
+  attribute :avg_stu_loan_debt
+  attribute :calendar
+  attribute :tuition_in_state
+  attribute :tuition_out_of_state
+  attribute :books
+  attribute :online_all
+  attribute :p911_tuition_fees
+  attribute :p911_recipients
+  attribute :p911_yellow_ribbon
+  attribute :p911_yr_recipients
+  attribute :accredited
+  attribute :accreditation_type
+  attribute :accreditation_status
+  attribute :caution_flag, if: -> { object.caution_flag.present? }
+  attribute :caution_flag_reason, if: -> { object.caution_flag.present? }
```

Comment:
Because JSON conforms to an object. And object has attributes that are either present or null, but the key should always be present or else its not the same object every-time.